keps/0001a-meta-kep-implementation.md

```diff
@@ -0,0 +1,133 @@
+---
+kep-number: 1a
+title: Meta KEP Implementation
+authors:
+  - "@justaugustus"
+  - "@calebamiles"
+  - "@jdumars"
+owning-sig: sig-pm
+participating-sigs:
+  - sig-architecture
+reviewers:
+  - "@idvoretskyi"
+  - "@timothysc"
+approvers:
+  - "@bgrant0607"
+  - "@jbeda"
+editor:
+creation-date: 2018-09-08
+last-updated: 2018-09-16
+status: provisional
+see-also:
+  - KEP-1
+replaces:
+superseded-by:
+---
+
+# Meta KEP Implementation
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Why another KEP?](#why-another-kep)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [Implementation Details / Notes / Constraints](#implementation-details--notes--constraints)
+    - [Define](#define)
+    - [Organize](#organize)
+    - [Visibility and Automation](#visibility-and-automation)
+    - [Constraints](#constraints)
+- [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+
+## Summary
+
+Drive KEP adoption through improved process, documentation, visibility, and automation.
+
+## Motivation
+
+The KEP process is the standardized structure for proposing changes to the Kubernetes project.
+
+In order to graduate KEPs to GA, we must iterate over the implementation details.
+
+This KEP seeks to define actionable / delegable items to move the process forward.
+
+Finally, by submitting a KEP, we gain an opportunity to dogfood the process and further identify areas for improvement.
+
+### Why another KEP?
+
+As [KEP-1] is currently de facto for the project, we must be careful to make changes to it in an iterative and atomic fashion.
+
+When proposing a KEP, we action over some unit of work, usually some area of code.
+
+In this instance, we treat [KEP-1] as the unit of work. That said, this would be considered a meta-KEP of the meta-KEP.
+
+### Non-Goals
+
+- API Review process
+- Feature request triage
+- Developer guide
+
+## Proposal
+
+### Implementation Details / Notes / Constraints
+
+#### Define
+
+- Refine existing KEP documentation
+- Define KEP [DACI]
+
+#### Organize
+
+- Move KEPs into [k/features]
```

Comment:
Can you say more about how you expect end users to use the new repository structure?

If I am an end user who wants to find out about the design of Kubernetes, what URL should I start at?  For Python, there is one page, called https://www.python.org/dev/peps/ that lists all their PEPs (~=KEPs).

Reply:
End users should be able to hit https://keps.k8s.io, which would be a redesigned version of https://contributor.kubernetes.io/keps/.

I envision this site / repo having at least three directories:
- `keps/` (KEPs)
- `design-proposals/` (historical design proposals from https://git.k8s.io/community/contributors/design-proposals)
- `arch[itecture]|design/` (design principles of Kubernetes, derived from reorganizing https://git.k8s.io/community/contributors/devel, mentioned [here](https://github.com/kubernetes/community/issues/2565#issuecomment-419185591))